py/desispec/io/meta.py

```diff
@@ -130,7 +130,7 @@ def get_raw_files(filetype, night, expid, rawdata_dir=None):
             guaranteed to match the regular expression [brz][0-9].
     """
     glob_pattern = findfile(filetype, night, expid, camera='*', rawdata_dir=rawdata_dir)
-    literals = map(re.escape,glob_pattern.split('*'))
+    literals = list(map(re.escape,glob_pattern.split('*')))
```

Comment:
Minor point: This could also be written as a list comprehension: `[re.escape(f) for f in glob_pattern.split('*')]`.